platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -1,31 +1,40 @@
+import { InstanceMetadata } from '../../types';
 import { PubSubService } from '../_shared/pubSubServiceInterface';
 import EVENTS from './EVENTS';
 
-const displaySetCache = [];
+const displaySetCache = new Map();
```

Comment:
Does it make sense to specify the type Generic parameters here? (i.e. `new Map<string, DisplaySet>`)?